packages/atlas/src/components/_buttons/CopyAddressButton/CopyAddressButton.tsx

```diff
@@ -20,25 +19,25 @@ export const CopyAddressButton: FC<CopyAddressButtonProps> = ({ address, classNa
     if (!address) {
       return
     }
-    copyToClipboard(address, 'Account address copied to clipboard')
+    copyToClipboard(address)
     setCopyButtonClicked(true)
     setTimeout(() => {
       setCopyButtonClicked(false)
     }, 3000)
```

Comment:
A small thing, but according to the design icon should change after 2 seconds. Also, it's a little buggy. Sometimes the text changes immediately, maybe we should move the timeout to useEffect?  Tooltips like acting weird in our app, if it's hard to fix, let's leave it as it is.

Reply:
I'll just block onClick for the duration of the timeout